packages/dev/core/src/PostProcesses/postProcess.ts

```diff
@@ -832,6 +838,7 @@ export class PostProcess {
             },
             this._engine
         );
+        this.onEffectCreatedObservable.notifyObservers(this._drawWrapper.effect);

```

Comment:
need to clear the observable on dispose else all good!

Reply:
Done :-)